release.yaml

```diff
@@ -1,20 +1,15 @@
 fleet:
```

Comment:
Also removed fleet because it was released previously as a forward-ported chart, but was not removed here.